bots/image-upload

```diff
@@ -59,17 +77,21 @@ def upload(store, source):
     return ret
 
 def main():
-    parser = argparse.ArgumentParser(description='Upload virtual machine images')
-    parser.add_argument("--store", default=UPLOAD, help="Where to send images")
+    parser = argparse.ArgumentParser(description='Upload bot state or images')
+    parser.add_argument("--store", default=UPLOAD, help="Where to send state or images")
+    parser.add_argument("--state", action="store_true", help="Loose images or state not recorded in git")
```

Comment:
"Loose" sounds odd here - does that mean that they get removed locally? That would be weird for an image/state upload command.

Reply:
The Loose means they're not recorded in git. Lets just remove that word.